bucket/luarocks.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": "3.8.0",
+    "description": "The package manager for Lua modules",
+    "homepage": "https://luarocks.org",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://luarocks.github.io/luarocks/releases/luarocks-3.8.0-windows-64.zip",
+            "hash": "d0bef92a9637df25bb86bad7ee3ae72e62a20a7b566d6f9cea175db0ebaecfd7",
+            "extract_dir": "luarocks-3.8.0-windows-64"
+        },
+        "32bit": {
+            "url": "https://luarocks.github.io/luarocks/releases/luarocks-3.8.0-windows-32.zip",
+            "hash": "017b606287f3192112e51e59333c91729be3e9bb9be0b84548d74903a620706a",
+            "extract_dir": "luarocks-3.8.0-windows-32"
+        }
+    },
+    "pre_install": [
+        "$file = 'config.lua'",
+        "$rocks = \"$(versiondir 'luarocks' 'current')\\rocks\" -replace \"\\\\\", \"/\"",
+        "if (!(Test-Path \"$persist_dir\\$file\")) {",
+        "   Write-Host 'File' $file 'does not exists. Creating.' -f Yellow",
+        "   $CONT = @(\"rocks_trees = { `\"$rocks`\" }\")",
+        "   Set-Content \"$dir\\$file\" ($CONT -join \"`r`n\") -Encoding Ascii",
+        "}"
+    ],
+    "bin": [
+        "luarocks.exe",
+        "luarocks-admin.exe"
+    ],
+    "env_add_path": "rocks\\bin",
```

Comment:
I downloaded the archive from https://luarocks.github.io/luarocks/releases/luarocks-3.8.0-windows-64.zip but it does not contain this folder. When is it created?

Reply:
When you add rocks and if it create bin.

I found this manifest needs more config and test... So convert to draft now